### PR TITLE
Update documentation to work without ncdev

### PR DESCRIPTION
**Pull request by nickvergessen**
ncdev has been killed and replaced with git + https://apps.nextcloud.com/developer/apps/generate

Please review @MariusBluem @BernhardPosselt

### CODE REVIEW

File: developer_manual/app/tutorial.rst [hunk 1]
```diff
@@ -9,11 +9,13 @@ This tutorial will outline how to create a very simple notes app. The finished a
 
 Setup
 =====
-After the `development tool <https://github.com/nextcloud/ncdev/blob/master/README.rst#installation>`_ has been installed the :doc:`development environment needs to be set up <../general/devenv>`. This can be done by either `downloading the zip from the website <https://nextcloud.com/install/>`_ or cloning it directly from GitHub::
+First the :doc:`development environment <../general/devenv>` needs to be set up. This can be done by either `downloading the zip from the website <https://nextcloud.com/install/>`_ or cloning it directly from GitHub::
 
-    ncdev setup core --dir nextcloud  --branch $BRANCH
+   git clone git@github.com:nextcloud/server.git --branch $BRANCH
+   git submodule init
+   git submodule update
```
Comment (MorrisJobke) [inline developer_manual/app/tutorial.rst]: git submodule update --init

File: developer_manual/general/devenv.rst [hunk 2]
```diff
@@ -48,15 +48,15 @@ Check out the code
...
 After the development tool installation make the directory writable::
 
   sudo chmod o+rw /var/www
 
 Then install Nextcloud from git::
 
-  ncdev setup --dir /var/www/<folder> server
+  git clone git@github.com:nextcloud/server.git --branch $BRANCH /var/www/<folder>
+  git submodule init
+  git submodule update
```
Comment (MorrisJobke) [inline developer_manual/general/devenv.rst]: git submodule update --init

**Comment by nickvergessen** [comment]
Updated

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
stable10: 7622640 0b317e7 6717932

**Comment by MorrisJobke** [comment]
stable9: 7a17520 052eb07 51de01a

**Pull request merged**